server/catgenome/build.gradle

```diff
@@ -160,9 +160,9 @@ dependencies {
 
     // Azure Storage Blob
     compile group: "com.azure", name: "azure-storage-blob", version: "12.14.0"
+    compile group: "com.azure", name: "azure-identity", version: "1.4.4"
```

Comment:
I've checked code once again and it seems that this new dependency brings a transitive dependency `net.minidev:json-smart` which breaks NGB json parsing configuration. Exclusion of this dependency helps to fix configuration and tests, but I cannot verify whether all azure credentials options work after such change, although Azure tests pass. Could you please check whether new code works normally with such build configuration:
```
 compile("com.azure:azure-identity:1.4.4") {
        exclude group: 'net.minidev', module: 'json-smart'
}
```